### PR TITLE
Add reverse hostname to failover IP

### PR DESCRIPTION
**Pull request by nclandrei**
I added in the new reverse hostname to the failover IP resource. Please let me know if I should change anything or if I did not follow the right path!

Again, please accept my apologies for such a long delay.

### CODE REVIEW

File: online/failover.go [hunk 1]
```diff
@@ -41,3 +41,16 @@ func (c *client) DeleteMACFailoverIP(address string) error {
 
 	return nil
 }
+
+func (c *client) SetReverseFailoverIP(address, reverse string) error {
+	target := fmt.Sprintf("%s/server/edit", serverEndPoint)
```
Comment (meyskens) [inline online/failover.go]: `/server/ip/edit` would be the right endpoint here

File: online/failover.go [hunk 2]
```diff
@@ -41,3 +41,16 @@ func (c *client) DeleteMACFailoverIP(address string) error {
...
+	_, err := c.doPOST(target, map[string]string{
+		"ip":      address,
```
Comment (meyskens) [inline online/failover.go]:
```suggestion
		"address":      address,
```

**Review by meyskens — CHANGES_REQUESTED** [review]
Also needs a delete action 
```
POST | reverse | string | Reverse (use the value 'false' to revert to the default reverse)
```

File: online/failover.go (hunk 1 above)
Comment (nclandrei) [reply online/failover.go]: just changed, sorry about this slip!

**Review by nclandrei — COMMENTED** [review]
Thanks for the comments and sorry for slipping some stuff.

File: online/failover.go (hunk 1 above)
Comment (meyskens) [reply online/failover.go]: Seems to be the same here :confused:

Comment (nclandrei) [reply online/failover.go]: Strange, but now it should be added in properly 😁

**Review by meyskens — APPROVED** [review]

**Pull request closed without merging**